src/Type/Definition/FieldArgument.php

```diff
@@ -77,12 +80,9 @@ public static function createMap(array $config) : array
         return $map;
     }
 
-    /**
-     * @return InputType&Type
```

Comment:
This should stay, no?

Reply:
I'd like that, but without some kind of assert my only other option is a cast, which sort of defeats the point of checking.

Reply:
so can we assert / ignore in static analysis for now? This can be IMO resolved better through templates later

Reply:
My next PR will be a full native assert overhaul across the board. You can consider that the price of merging this PR.

Reply:
Why do you need to remove this, though? As a user of this library + PHPStan, i would appreciate the precise type, event if there is a slim chance that it might be wrong (not sure how that might even happen).

Adding an `assert()` later seems like a great idea, too - i think we can have both.

Reply:
> Why do you need to remove this, though?

Because if I restore it, it conflicts with the known return type of `Schema::resolveType`, which is `Type`. I believe the best way to make everybody happy is going to be with native asserts, but setting those up properly will involve more work and more discussion, and I really think it warrants a dedicated PR.

Reply:
TBH the type doc can be IMO kept as is and the type incompatibility within this method ignored until the next PR